spec.html

```diff
@@ -42121,15 +42122,15 @@ <h2>Syntax</h2>
 
         FourToSeven :: one of
           `4` `5` `6` `7`
+
+        NonOctalDecimalEscapeSequence :: one of
```

Comment:
This could just be an alias for `NonOctalDigit`, but I think SV would be less clean to specify that way? Feel free to correct me though.